models/demos/deepseek_v3/tests/test_model.py

```diff
@@ -127,11 +134,33 @@ def test_forward_pass(
     paged_input_caches, torch_page_tables = paged_caches_from_torch(input_cache, dp_factor, paged_config, user_id)
 
     # Set up model config
-    weight_config = Model1D.convert_weights(hf_config_short, [state_dict], tmp_path, mesh_device)
+    weights_type = "real_weights" if use_real_weights else "random_weights"
```

Comment:
I will soon be merging a PR with the caching functionality, so let's do away without caching for now.